core/src/main/scala/kafka/common/InterBrokerSendThread.scala

```diff
@@ -78,6 +77,8 @@ abstract class InterBrokerSendThread(
       failExpiredRequests(now)
       unsentRequests.clean()
     } catch {
+      case _: DisconnectException if !networkClient.active() =>
+        // Other threads can shutdown network client by calling InterBrokerSendThread#shutdown
```

Comment:
Could we also state the `DisconnectException` is expected when `NetworkClient#initiateClose` is called?

Reply:
done